tests/LockingWrapperTest.php

```diff
@@ -33,13 +33,6 @@ protected function setup() {
 		parent::setUp();
 
 		App::checkAppEnabled('files_locking');
-		OC_User::clearBackends();
```

Comment:
Tests ran fine on my end without these, and $this->user was never even used in these tests...